### PR TITLE
Initialised Frontend

### PR DESCRIPTION
**Pull request by karanveer962**

### CODE REVIEW

**Review by karanveer962 — COMMENTED** [review]
Checked Correctly.

File: client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
```
Comment (kanha638) [inline client/.gitignore:6]: why this is here

Comment (kanha638) [reply client/.gitignore:6]: @karanveer962

**Review by kanha638 — APPROVED** [review]
Changed

**Pull request merged**